iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -533,7 +539,14 @@ private void TwinMessageListener(Twin twin, string correlationId, TwinCollection
                     TaskCompletionSource<Twin> task;
                     if (_twinResponseCompletions.TryRemove(correlationId, out task))
                     {
-                        task.SetResult(twin);
+                        if(ex == default)
```

Comment:
missed a space after `if`

Reply:
I'll take care of this in a follow up PR.